lane.py

```diff
@@ -118,17 +124,24 @@ def navigate(self, entrypoint):
 
     @classmethod
     def from_request(
-            cls, library, facet_config, get_argument, worklist, **extra_kwargs
+            cls, library, facet_config, get_argument, get_header, worklist,
+            **extra_kwargs
     ):
         """Load a faceting object from an HTTP request.
 
+        # TODO: Is the 'library' argument redundant?
+
```

Comment:
I think it's not, because certain feeds need a custom facet setup instead of using the library's defaults (e.g. a series feed can order by series position).